gapis/api/vulkan/framegraph.go

```diff
@@ -69,17 +99,14 @@ func newAttachmentInfo(desc VkAttachmentDescription, imgView ImageViewObjectʳ,
 		loadOp = desc.LoadOp()
 		storeOp = desc.StoreOp()
 	}
+	imgObj := imgView.Image()
```

Comment:
Very nit: You probably don't need the variable.

Reply:
Turns out you do, because Go won't let you do `&(imgView.Image())` (hence this new variable here).